CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### Breaking changes
 
+- Removed support for `primaryButtonColor` field on `initPaymentSheet()`. Please use the new `appearance` field instead. [#940](https://github.com/stripe/stripe-react-native/pull/940)
```

Comment:
cc @yuki-stripe we didn't do this on Android and iOS. Is this something we are ok with doing?

Reply:
Yeah I think we want to do this eventually as well.  We can and should give users the specific API they should use instead though e.g.`appearance.primaryButton.colors.primary` (or whatever it is).

Reply:
addressed in https://github.com/stripe/stripe-react-native/pull/940/commits/fba743881128ce0a5124b84aa33a29e3a6225e95